spec/rspec/core/formatters/exception_presenter_spec.rb

```diff
@@ -211,8 +224,9 @@ module RSpec::Core
 
       it 'wont produce a stack error when the cause is an older exception', :if => RSpec::Support::RubyFeatures.supports_exception_cause? do
         allow(the_exception).to receive(:cause) do
```

Comment:
Is this needed? `the_exception` and `second_exception` point fingers at each other already.

Reply:
Yes, this is a loop. The others don't loop. (the_exception => second_exception => first_exception => nil)

Reply:
Oh, I've mixed `first_exception` with `the_exception` in `second_exception`'s cause.
👍